be/src/common/config.h

```diff
@@ -179,9 +179,9 @@ CONF_mInt32(status_report_interval, "5");
 // Local directory to copy UDF libraries from HDFS into.
 CONF_String(local_library_dir, "${UDF_RUNTIME_DIR}");
 // Number of olap/external scanner thread pool size.
-CONF_mInt32(doris_scanner_thread_pool_thread_num, "48");
+CONF_mInt32(scanner_thread_pool_thread_num, "48");
```

Comment:
The change of these should be compatible. Because if the user use the old configuration, the start of be will failed.

Reply:
i did this when Properties::load from configure file check and replace the related config name